features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/PrivacyOverrideExtensions.kt

```diff
@@ -22,3 +22,31 @@ fun View.datadogSessionReplayHidden(hide: Boolean) {
         this.setTag(R.id.datadog_hidden, null)
     }
 }
+
+/**
+ * Allows overriding the image privacy for a view in Session Replay.
+ *
+ * @param privacy the new privacy level to use for the view
+ * or null to remove the override.
+ */
+fun View.datadogSessionReplayImagePrivacy(privacy: ImagePrivacy?) {
```

Comment:
> [!NOTE]
> Shouldn't we use a verb for function name, like `setDatadogSessionReplayImagePrivacy` (or just `setSessionReplayImagePrivacy` or `setDatadogImagePrivacy`) ?

Reply:
I've renamed the apis to align with our discussion